src/lndHealth.ts

```diff
@@ -0,0 +1,47 @@
+import { authenticatedLndGrpc, getWalletInfo } from 'lightning';
+import _ from "lodash";
+import { baseLogger } from "./logger";
+import { ILndParamsAuthed } from "./lndAuth"
+
+const refresh_time = 5000 // ms
+
+// TODO replace with
+// const asyncForever = require('async/forever');
+const loop = async ({socket, params}: {socket: string, params: ILndParamsAuthed}) => {
```

Comment:
Can we not just call isUp inside the setTimeout to remove the need for recursion? Maybe I'm missing why there's a necessity for recursion here